cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,18 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#/login');
 
+    cy.get(':nth-child(1) > .form-control')
+      .type('QA_test@gmail.com');
+
```

Comment:
extra empty lines are not really needed inside test(it) https://mate-academy.github.io/qa-program/checklists/cypress.html